taniumrest_connector.py

```diff
@@ -642,34 +671,39 @@ def _determine_num_results_complete(self, data):
         return num_complete, num_incomplete
 
     def _question_result(self, timeout_seconds, results_percentage, endpoint, action_result,
-                         wait_for_results_processing=None, return_when_n_results_available=None, wait_for_n_results_available=None):
+                         wait_for_results_processing=None, return_when_n_results_available=None,
+                         wait_for_n_results_available=None):
 
-        max_range = int(timeout_seconds / WAIT_SECONDS) + (1 if timeout_seconds % WAIT_SECONDS == 0 else 2)
+        max_range = int(timeout_seconds / TANIUMREST_WAIT_SECONDS) + \
+            (1 if timeout_seconds % TANIUMREST_WAIT_SECONDS == 0 else 2)
```

Comment:
suggestion: similar to the condition above. What do you think about using brackets to work around new line here?
```python
max_range = (int(timeout_seconds / TANIUMREST_WAIT_SECONDS) + 
            (1 if timeout_seconds % TANIUMREST_WAIT_SECONDS == 0 else 2))
```
It might be a bit easier to read later on.

Reply:
Similar to above, Removed the line break. Because, earlier the max line length was 130, but now it is increased to 145. So, no need for a line break.